src/Dialog/dialog.scss

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright (c) 2021 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-4-Clause
+ */
+
+@import '../variables';
```

Comment:
Suggestion: create a regular `.css` file instead. Don't remember exactly why, but there was a proposition to use regular `.css` files whenever we could.

You don't even need the import in a regular `.css` file, believe you'll just find the variables runtime.